packages/core/src/command/consolidated/get-snapshot-local.ts

```diff
@@ -125,3 +133,25 @@ async function postSnapshotToApi({
     }
   );
 }
+
+export function removeContainedPatients(bundle: Bundle): Bundle {
+  if (!bundle.entry) return bundle;
+
+  const updatedEntry = bundle.entry.map(entry => {
+    if (entry.resource && "contained" in entry.resource) {
+      return {
+        ...entry,
+        resource: {
+          ...entry.resource,
+          contained: entry.resource.contained?.filter(r => !isPatient(r)),
```

Comment:
We also want to remove the correct patient from contained? Is this potentially editing what the CX wants to do?